Project structure/Gradle build system.md

```diff
@@ -176,8 +176,8 @@ Each (build type, product flavor) combination is called a **build variant**.
 Projects with no flavors still have build variants, but the single default flavor is used, nameless, making the list of variants the same as the list of build types.
 To build the variant you want, you should select it from the AS menu on the left side.
 
-![Build variant menu](/img/build_variant_1.png "Build variant menu")
-![Build variant selection](/img/build_variant_2.png "Build variant selection")
+<img src="/img/build_variant_1.png" alt="Build variant menu" width="80%" height="75%">
+<img src="/img/build_variant_2.png" alt="Build variant selection" width="80%" height="75%">
```

Comment:
Why scale down the screenshots? 😄 does it look better on the web now that they're so big?

Reply:
I took 4k screenshots and on my preview they were huge 😅, will merge like this in staging and then I will see how it looks on beta.infinum